ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/tree/node/services/TreeNodeConverterImpl.java

```diff
@@ -51,22 +56,23 @@ private List<Statement> addTreeNodeStatements(TreeNode treeNode) {
 		statements.add(createStatement(currentFragmentId, RDF_SYNTAX_TYPE, createResource(TREE_NODE_RESOURCE)));
 		statements.addAll(relationStatementConverter.getRelationStatements(treeNode.getRelations(), currentFragmentId));
 
-		addLdesCollectionStatements(statements, treeNode.isView(), currentFragmentId);
+		addLdesCollectionStatements(statements, treeNode.isView(),
+				ldesConfig.getHostName() + "/" + ldesConfig.getCollectionName() + treeNode.getFragmentId());
```

Comment:
Kan je van deze String een variabele maken zodat dit altijd hetzelfde is als lijn 53-54? Zo voorkomen we dat er ooit 1 van de 2 wordt aangepast en dat we ineens twee verschillende resources hebben